lib/editor/components/timetable/TimetableGrid.js

```diff
@@ -131,38 +254,32 @@ export default class TimetableGrid extends Component {
         // retain active focus on the grid
         this._focusOnGrid()
         break
+      case 74: // j
       case 40: // down
-        // check if done with command key or ctrl
-        if (evt.metaKey || evt.ctrlKey) {
+        if (ctrlPressed) {
           // prevent default up behavior
           evt.preventDefault()
           // override ArrowKeyStepper
           evt.stopPropagation()
 
           // move all the way to the bottom
           updateScroll(data.length - 1, scrollToColumn)
-
-          // retain active focus on the grid
-          this._focusOnGrid()
-        }
-        break
-      case 67:
-        // handle copy
-        if (evt.ctrlKey) {
-          console.log('copy pasta')
+        } else {
+          this.offsetScrollRow(1)
         }
-        return false
+        // retain active focus on the grid
+        return this._focusOnGrid()
       default:
         break
     }
-    // input was 0-9
-    if (!activeCell && evt.keyCode >= 48 && evt.keyCode <= 57 && !evt.ctrlKey) {
-      setActiveCell(`${scrollToRow}-${scrollToColumn}`)
-    }
-    // input was a-z
-    if (!activeCell && evt.keyCode >= 65 && evt.keyCode <= 90 && !evt.ctrlKey) {
-      setActiveCell(`${scrollToRow}-${scrollToColumn}`)
-    }
+    // // input was 0-9
+    // if (!activeCell && evt.keyCode >= 48 && evt.keyCode <= 57 && !ctrlPressed) {
+    //   setActiveCell(`${scrollToRow}-${scrollToColumn}`)
+    // }
+    // // input was a-z
+    // if (!activeCell && evt.keyCode >= 65 && evt.keyCode <= 90 && !ctrlPressed) {
+    //   setActiveCell(`${scrollToRow}-${scrollToColumn}`)
+    // }
```

Comment:
Let's just remove these commented out lines completely.  It's in the commit history if we need to copy it back.